climada/hazard/test/test_trop_cyclone.py

```diff
@@ -395,21 +427,21 @@ def test_holland_2010_pass(self):
         hol_x = _x_holland_2010(si_track, d_centr, close_centr)
         np.testing.assert_array_almost_equal(hol_x, [
             [0.5, 0.500000, 0.485077, 0.476844, 0.457291],
-            [0.5, 0.500000, 0.410997, 0.289203, 0.000000],
-            [0.5, 0.497620, 0.131072, 0.000000, 0.000000],
+            [0.5, 0.500000, 0.410997, 0.400000, 0.000000],
+            [0.5, 0.497620, 0.400000, 0.400000, 0.400000],
             [0.5, 0.505022, 1.403952, 1.554611, 2.317948],
         ])
 
         v_ang_norm = _stat_holland_2010(si_track, d_centr, close_centr, hol_x)
-        np.testing.assert_array_almost_equal(v_ang_norm, [
+        np.testing.assert_allclose(v_ang_norm, [
             # first column: converge to 0 when approaching storm eye
             # second column: vmax at RMW
             # fourth column: peripheral speed (17 m/s) at peripheral radius (unless x is clipped!)
             [0.0000000, 35.000000, 21.181497, 17.00000, 12.103461],
-            [1.2964800, 34.990037, 21.593755, 17.00000, 0.0000000],
-            [0.3219518, 15.997500, 13.585498, 16.00000, 16.000000],
+            [1.2964800, 34.990037, 21.593755, 12.89131, 0.0000000],
+            [0.3219518, 15.997500, 9.7120060, 8.087240, 6.2289690],
             [24.823469, 89.992938, 24.381965, 17.00000, 1.9292020],
-        ])
+        ], atol=1e-5)
```

Comment:
These values are given with a precision of 1e-6. So why is it necessary to set the absolute tolerance to 1e-5? Doesn't this mean there is a computation error in here somewhere?

Reply:
Due to formatting some values were given with more or less precision. I reformatted to provide all numbers with 1e-6 precision and then the atol can be set accordingly.